src/Umbraco.Web.UI/Umbraco/config/lang/da.xml

```diff
@@ -264,6 +264,8 @@
     <key alias="titleOptional">Titel (valgfri)</key>
     <key alias="altTextOptional">Alternativ tekst (valgfri)</key>
     <key alias="type">Type</key>
+    <key alias="variantsToPublish">Hvilke varianter vil du publicere?</key>

```

Comment:
The other Danish labels in the say context says _udgive_, so could you use that as well here?

```suggestion
    <key alias="variantsToPublish">Hvilke varianter vil du udgive?</key>

```